x11-wm/openbox/openbox-3.6.1-r3.ebuild

```diff
@@ -46,9 +51,8 @@ RDEPEND="dev-libs/glib:2
 		')
 	)
 "
-DEPEND="${RDEPEND}
-	sys-devel/gettext
-	virtual/pkgconfig
+DEPEND="
+	${RDEPEND}
```

Comment:
I prefer `${VAR}` inherits without line break (as it was before) but that's just how kde proj convention taught me.

Reply:
I was a little indecisive about it, but ended up with it feeling more consistent with the rest, though it somehow makes sense to me both ways.

I'll keep that in mind for the next time.

As always, many thanks!